app/ui/scalatags.scala

```diff
@@ -55,6 +55,7 @@ trait ScalatagsSnippets extends Cap {
   val goodTag                                = tag("good")
   val badTag                                 = tag("bad")
   val timeTag                                = tag("time")
+  val dialog                                 = tag("dialog")
```

Comment:
Actually, since the build went through, I don' think that tag was actually used.

Reply:
Indeed, currently it's not used (that's why I missed it), but I thought that anyway it's not a good idea to delete something not related to the issue.